app/controllers/EPayeStartController.scala

```diff
@@ -74,14 +76,14 @@ class EPayeStartController @Inject() (
 
   def routeResponse(data: EligibilityData)(implicit R: Request[_]): Result = {
     if (data.hasRejections) {
-      val leadingContentToShow: Html = data.rejections.head match {
```

Comment:
This is where the meat and potatoes of the change are, along with new template, will comment there too.